src/views/Profile/components/ApproveConfirmButtons.tsx

```diff
@@ -17,6 +17,7 @@ interface ApproveConfirmButtonsProps {
   onConfirm: () => void
   buttonArrangement?: ButtonArrangement
   confirmLabel?: string
+  confirmId?: string
```

Comment:
Why not use the native `id` prop already there?

Reply:
Ah - because the `ApproveConfirmButtons` have multiple buttons within them - an approve, and a confirm. This id is only passed to the confirm buttons